docs/source/sources.txt

```diff
@@ -0,0 +1,401 @@
+retworkx.adjacency_matrix.html
```

Comment:
This file seems autogenerated, should it be included in the source tree or not?

Reply:
It's needed in the tree. There's an ordering problem we don't know what the paths were for `stubs/` were (or what the apiref paths are) so we can't statically build a list of redirects in the configuration. This is going to just be a static list for redirects from old versions of the documentation and it won't ever grow. It's just so we know what to add a html redirect page for at the end of the doc build.

Reply:
I'm good with a static list, that should be on the tree indeed